README.md

```diff
@@ -147,8 +147,8 @@ USAGE
   [--loglevel trace|debug|info|warn|error|fatal|TRACE|DEBUG|INFO|WARN|ERROR|FATAL]
 
 OPTIONS
-  -d, --outputdir=outputdir                                                         [default: ~/.commerce
-                                                                                    /force-app] Directory to output the
+  -d, --outputdir=outputdir                                                         [default: ~/.commerc
```

Comment:
should this be `~/.commerce`?

Reply:
It is still .commerce I think the formatting just screwed up here. I didn't change this file manually